### PR TITLE
Improve accessibility

### PR DESCRIPTION
**Pull request by psjnek**

### CODE REVIEW

File: _layouts/default.html
Comment (lzh-ableton) [inline _layouts/default.html:1]: Please add `lang="en"` to the `html` tag.

File: _layouts/link.html
Comment (lzh-ableton) [inline _layouts/link.html:1]: Please add `lang="en"` to the `html` tag.

File: _layouts/linkkit.html
Comment (lzh-ableton) [inline _layouts/linkkit.html:1]: Please add `lang="en"` to the `html` tag.

File: export/index.md [hunk 1]
```diff
@@ -1,4 +1,5 @@
 ---
+lang: en-US
```
Comment (lzh-ableton) [inline export/index.md]: Please remove this line.

File: index.md [hunk 2]
```diff
@@ -1,4 +1,5 @@
 ---
+lang: en-US
```
Comment (lzh-ableton) [inline index.md]: Please remove this line.

File: link/index.md [hunk 3]
```diff
@@ -1,4 +1,5 @@
 ---
+lang: en-US
```
Comment (lzh-ableton) [inline link/index.md]: Please remove this line.

File: linkkit/index.md [hunk 4]
```diff
@@ -1,4 +1,5 @@
 ---
+lang: en-US
```
Comment (lzh-ableton) [inline linkkit/index.md]: Please remove this line.

**Review by lzh-ableton — CHANGES_REQUESTED** [review]
Hi @WebSnke. Thank you for your PR! 

I have a few suggestions to improve the `lang` attribute implementation. I'll explain them here and then leave comments on each file to change.

1. Instead of `en-US`, we can use `en`. This is because in general, shorter language tags are preferred. (See [W3: Language tags in HTML and XML](https://www.w3.org/International/articles/language-tags/#overview): _The golden rule when creating language tags is to keep the tag as short as possible. Avoid region, script or other subtags except where they add useful distinguishing information._)

2. Since our site is only in English, we can set the `lang` attribute directly in the `html` tag of layout files (which looks like was your original approach). This avoids the overhead of adding the property individually to all .md files, an approach which would be more useful if we had multiple languages to support.

Other than those changes, before we can merge your PR, we need a signed CLA. You can find all the information on https://ableton.github.io/cla/

Thanks and let me know if you have any questions!

**Comment by psjnek** [comment]
> Hi @WebSnke. Thank you for your PR!
> 
> I have a few suggestions to improve the `lang` attribute implementation. I'll explain them here and then leave comments on each file to change.
> 
>     1. Instead of `en-US`, we can use `en`. This is because in general, shorter language tags are preferred. (See [W3: Language tags in HTML and XML](https://www.w3.org/International/articles/language-tags/#overview): _The golden rule when creating language tags is to keep the tag as short as possible. Avoid region, script or other subtags except where they add useful distinguishing information._)
> 
>     2. Since our site is only in English, we can set the `lang` attribute directly in the `html` tag of layout files (which looks like was your original approach). This avoids the overhead of adding the property individually to all .md files, an approach which would be more useful if we had multiple languages to support.
> 
> 
> Other than those changes, before we can merge your PR, we need a signed CLA. You can find all the information on https://ableton.github.io/cla/
> 
> Thanks and let me know if you have any questions!

1. I used the en-us because specififying the accent is helpful for screen readers.
2. Sure, I can revert the changes.

**Comment by psjnek** [comment]
I also signed the CLA already.

**Comment by lzh-ableton** [comment]
> 1. I used the en-us because specififying the accent is helpful for screen readers.

In our case, our content being spoken in a US dialect isn't a necessity, and regardless, region subtags are typically ignored (source: [WebAIM: Document and Content Language](https://webaim.org/techniques/language/#region)). We also use `en` on ableton.com, rather than `en-US`. If you could change this then the PR would be good to go!

> I also signed the CLA already.

Great, thanks!

**Review by lzh-ableton — APPROVED** [review]
LGTM! Thank you for your contribution @WebSnke!

@nre-ableton This could be merged when you have the chance.

**Comment by nre-ableton** [comment]
@WebSnke thanks for your contribution! 👍

I took the liberty of rebasing your branch to drop the "Move language attribute to Markdown files" commit and its corresponding revert.

**Review by nre-ableton — APPROVED** [review]

**Comment by psjnek** [comment]
Thanks. May I ask how you got your job at Ableton as a developer btw?

**Comment by nre-ableton** [comment]
> Thanks. May I ask how you got your job at Ableton as a developer btw?

Keep your eye on https://www.ableton.com/jobs 🙂

**Pull request merged**